tests/jeos/image_info.pm

```diff
@@ -39,7 +39,7 @@ sub check_db {
     }
 
     # CASEDIR var is always set, and if when no specified, the value is "sle" for OSD and "opensuse" for O3
-    return if (get_required_var('CASEDIR') !~ m/^sle$|^opensuse$/);
+    return 0 if (get_required_var('CASEDIR') !~ m/^sle$|^opensuse$/);
```

Comment:
Oh, returning `undef` was a problem before?

Reply:
Maybe not, but there is `return 0` in all other conditions, just to be precise, and just `return 1` if everything is ok. Some sort of "return True/False" for python :)